en/docs/develop/writing-a-custom-federated-authenticator.md

```diff
@@ -106,1262 +111,118 @@ public interface ApplicationAuthenticator extends Serializable {
-                Twitter twitter = (Twitter) request.getSession().getAttribute("twitter");
-                RequestToken requestToken = (RequestToken) request.getSession().getAttribute("requestToken");
-                String verifier = request.getParameter("oauth_verifier");
-                try {
-                    AccessToken token=twitter.getOAuthAccessToken(requestToken, verifier);
-                    request.getSession().removeAttribute("requestToken");
-                    User user= twitter.verifyCredentials();
-                    buildClaims(user,context);
-                } catch (TwitterException e) {
-                    LOGGER.error("Exception while obtaining OAuth token form Twitter", e);
-                    throw new AuthenticationFailedException("Exception while obtaining OAuth token form Twitter",e);
-                }
-        }
-            
-        public void buildClaims(User user, AuthenticationContext context) {
-            
-                    context.setSubject(String.valueOf(user.getId()));
-                    Map<ClaimMapping, String> claims = new HashMap<ClaimMapping, String>();
-                    claims.put(ClaimMapping.build("name", "name", null,false), user.getName());
-                    claims.put(ClaimMapping.build("screen_name", "screen_name", null,false), user.getScreenName());
-                    claims.put(ClaimMapping.build("url", "url", null,false), user.getURL());
-            
-                    context.setSubjectAttributes(claims);
-        }
-        ```
+1. Access the Management console of the partner identity server.
+2. Then go to the Service Providers under the main tab. 
+   Then add Service Provider Name and register it. 
+   Let’s use the playground app and refer to [this](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-playground2-webapp) to configure the playground app.
+3. Then List the Service Providers and edit the service provider by navigation to the** 
+   OAuth/OpenID Configuration** under **Inbound Authentication Configuration** and add 
+   [https://localhost:9443/commonauth](https://localhost:9443/commonauth) as the callback URL.
+4. Create a user **Alex** in the partner identity server.
 
-4.  The `          buildClaims         ` method saves the retrieved user
-    attributes to the authenticated context in the Identity Server. That
-    is needed to map the claims to the built in claims of IS.
-5.  After implementing these methods you can build your bundle. After
-    building it you have to put that into the
-    `          <IS_HOME>/repository/components/dropins         `
-    directory.
-6.  [Restart the Identity Server](../../setup/running-the-product) to use the
-    Twitter authenticator in IS.
+### Configure Federated Authenticator
 
-### Custom authenticator for Facebook
+To configure the federated authenticator, click the add button under **Identity Providers** and add an IDP name as 
```

Comment:
```suggestion
To configure the federated authenticator, click the **Add** button under **Identity Providers** and add the IDP name as 
```

Reply:
Fixed https://github.com/wso2/docs-is/pull/2527/commits/ffc80dd6ac9088892baa7c56e5f03a9aa0768d3f